### PR TITLE
Added content repository docs

### PR DESCRIPTION
**Pull request by wachterjohannes**
Main PR https://github.com/sulu-io/sulu/pull/1808

### CODE REVIEW

File: bundles/content/content-repository.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
+ContentRepository
+=================
+
+The content-repository was developed to query the content raw-data. The
+developer can decide which properties will be loaded.
+
+The main goal of the repository is to centralize the core features ghost, shadow
+and internal links. These things will be handled full automatically. If you are
+user this repository.
```
Comment (danrot) [inline bundles/content/content-repository.rst]: This sentence doesn't really make sense.

File: bundles/content/content-repository.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+Usage
+-----
+
+There are two ways to use it. One in the front-end over an rest-api and one with
+a backend service.
+
+**API**
```
Comment (danrot) [inline bundles/content/content-repository.rst]: I would use another heading here (same for service)

File: bundles/content/content-repository.rst [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+The mandatory parameters are:
+
+.. list-table::
+
+    * - Name
+      - Example
+      - Description
+    * - locale
+      - de
+      - Localisation which will be used.
```
Comment (danrot) [inline bundles/content/content-repository.rst]: `Localization`

File: bundles/content/content-repository.rst [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    * - Name
+      - Default
+      - Description
+    * - exclude-ghosts
+      - false
+      - If true ghost will be filtered.
+    * - exclude-shadows
+      - false
+      - If true shadows will be filtered.
+    * - mapping
```
Comment (danrot) [inline bundles/content/content-repository.rst]: Don't forget to adapt (also in the text)

File: bundles/content/content-repository.rst [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+    * - followInternalLink
+      - if this is false the link will not be resolved.
+    * - properties
+      - List of hydrated properties
+
+You can build this mapping over the mapping builder:
+
+.. code-block:: php
+
+        $mapping = Sulu\Component\Content\Repository\Mapping\MappingBuilder::create()
```
Comment (danrot) [inline bundles/content/content-repository.rst]: Since we never use FQNs in the code, I would also not do it here.

**Comment by wachterjohannes** [comment]
@danrot fixed

**Comment by wachterjohannes** [comment]
merged in develop

**Pull request closed without merging**